lib/solidus_usps/engine.rb

```diff
@@ -21,6 +21,7 @@ class Engine < Rails::Engine
       require 'solidus_usps/calculator/first_class_package_international'
       require 'solidus_usps/calculator/ground_advantage'
       require 'solidus_usps/calculator/media_mail'
+      require 'solidus_usps/calculator/media_mail_with_insurance'
```

Comment:
After rebasing against `main`, we should add this to the `"solidus_usps.register.calculators"` block too.